internal/api/handler.go

```diff
@@ -341,8 +346,17 @@ func (h *Handler) handleTokenExchange(c *OAuthAppContext, w http.ResponseWriter,
 
 	h.metricsCollector.IncGeneratedToken(c.OAuthApplication.Identifier)
 
+	expiry := token.Extra("expires_in").(float64)
+	tokenResponse := &TokenResponse{
+		ExpiresIn: expiry,
+	}
+	tokenResponse.AccessToken = token.AccessToken
+	tokenResponse.RefreshToken = token.RefreshToken
+	tokenResponse.TokenType = token.TokenType
+	tokenResponse.Expiry = token.Expiry
```

Comment:
This can be simplified:
```suggestion
	tokenResponse := &TokenResponse{
		ExpiresIn: expiry,
		Token:     *token,
	}
```